resources/config-defaults.edn

```diff
@@ -123,7 +123,9 @@
  :theme {:color1 "#eee"
          :color2 "#555"
          :color3 "#000"
-         :color4 "#000"}
+         :color4 "#000"
+         ;; navigation type options
+         :upper-left-logo false}
```

Comment:
How about we name this `:navbar-logo` because it may not always be in the upper or left?

Reply:
sounds good!